modules/integration/src/test/java/org/apache/synapse/samples/framework/tests/advanced/Sample657.java

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Year is missing in the license header?

Reply:
Apache license header included in wso2-synapse files doesn't contain year

Reply:
But some classes has. So then we might have to fix those.

Reply:
I think the new files are licensed under wso2 and the forked files from apache still have the apache license header. Correct me if I am mistaken.

Reply:
All the integration test classes have the same license header (doesn't contain year). So how are we going to proceed?